### PR TITLE
Bug 1056984 Remove external link icons from "Signature report" links

### PR DESCRIPTION
**Pull request by Tchanders**
Also add titles to links and italicise the link in the "Signature facet" tab

### CODE REVIEW

File: webapp-django/crashstats/supersearch/templates/supersearch/search_results.html [hunk 1]
```diff
@@ -90,22 +90,24 @@
...
+                                    _no_base=True,
+                                    signature=hit.term,
+                                    page=None,
+                                    _columns=None,
+                                    _facets=None
+                                ) }}"
+                            {% else %}
+                                href="{{ url('crashstats:report_list') }}?signature={{ hit.term | urlencode }}&amp;{{ report_list_query_string }}"
+                            {% endif %}
+                                title="Click to leave this search and go to this individual report"
```
Comment (AdrianGaudebert) [inline webapp-django/crashstats/supersearch/templates/supersearch/search_results.html]: Please change ``individual`` to ``signature`` here.

**Comment by AdrianGaudebert** [comment]
Could we use a different icon here in replacement of the "external" one? Something very light, just to show that those links will take you outside of the Search page. I'm open to suggestion, or to disagreement. :)

File: webapp-django/crashstats/supersearch/templates/supersearch/search_results.html (hunk 1 above)
Comment (Tchanders) [inline webapp-django/crashstats/supersearch/templates/supersearch/search_results.html]: OK done

**Comment by Tchanders** [comment]
Perhaps it would make more sense to add icons (e.g. a plus) to the left of the search terms, to indicate that they are not links, but filters?

**Comment by AdrianGaudebert** [comment]
That could be a nice thing indeed! I have filed a bug for that, since it's a bit out of scope here: https://bugzilla.mozilla.org/show_bug.cgi?id=1152243 (feel free to work on it :) )

r+

**Pull request merged**